### PR TITLE
Fix mismatch messaging during podman preset

### PR DESCRIPTION
**Pull request by praveenkumar**
Currently most of the places still have `Openshift cluster` as part of
string and show as message to user when perform different operation like
`delete`, `setup` ..etc. This PR tries to fix it by using preset friendly
message string and show it to user as per preset.

fixes: #2863

### CODE REVIEW

File: cmd/crc/cmd/delete.go [hunk 1]
```diff
@@ -38,7 +40,7 @@ func deleteMachine(client machine.Client, clearCache bool, cacheDir string, inte
 		if !interactive && !force {
 			return false, errors.New("non-interactive deletion requires --force")
 		}
-		yes := input.PromptUserForYesOrNo("Do you want to delete the OpenShift cluster cache", force)
+		yes := input.PromptUserForYesOrNo("Do you want to delete the cache", force)
```
Comment (cfergeau) [inline cmd/crc/cmd/delete.go:43]: "the cache used by CodeReady Containers"?

File: cmd/crc/cmd/delete.go [hunk 2]
```diff
@@ -52,7 +54,8 @@ func deleteMachine(client machine.Client, clearCache bool, cacheDir string, inte
 		return false, errors.New("non-interactive deletion requires --force")
 	}
 
-	yes := input.PromptUserForYesOrNo("Do you want to delete the OpenShift cluster", force)
+	yes := input.PromptUserForYesOrNo(fmt.Sprintf("Do you want to delete the %s", crcPreset.UserFriendlyName(crcConfig.GetPreset(config))),
+		force)
```
Comment (cfergeau) [inline cmd/crc/cmd/delete.go:58]: For what it's worth, building strings this way must not be done when you want to translate your application, "the" might require a different translation depending on what '%s' is replaced with . We are not translating crc, so this is mostly theoritical :)

File: cmd/crc/cmd/setup.go [hunk 3]
```diff
@@ -30,8 +31,8 @@ func init() {
 
 var setupCmd = &cobra.Command{
 	Use:   "setup",
-	Short: "Set up prerequisites for the OpenShift cluster",
-	Long:  "Set up local virtualization and networking infrastructure for the OpenShift cluster",
+	Short: "Set up prerequisites for the OpenShift cluster or Podman runtime",
+	Long:  "Set up local virtualization and networking infrastructure for the OpenShift cluster or Podman runtime",
```
Comment (cfergeau) [inline cmd/crc/cmd/setup.go:35]: `for using  CodeReady Containers` ?

File: cmd/crc/cmd/start.go [hunk 4]
```diff
@@ -35,21 +35,21 @@ func init() {
 	addOutputFormatFlag(startCmd)
 
 	flagSet := pflag.NewFlagSet("start", pflag.ExitOnError)
-	flagSet.StringP(crcConfig.Bundle, "b", constants.GetDefaultBundlePath(crcConfig.GetPreset(config)), "The system bundle used for deployment of the OpenShift cluster")
+	flagSet.StringP(crcConfig.Bundle, "b", constants.GetDefaultBundlePath(crcConfig.GetPreset(config)), "The system bundle used for provision virtual machine")
```
Comment (cfergeau) [inline cmd/crc/cmd/start.go:38]: `to provision the virtual machine`.

**Review by cfergeau — COMMENTED** [review]
Let's hold this until this has had some scrutiny from Kevin, I don't think it's a must-have for it.
It might be desirable to limit the use of `UserFriendlyName()` and to use more of the `virtual machine` or `CodeReady Containers instance` terminology.

File: cmd/crc/cmd/start.go (hunk 4 above)
Comment (gbraad) [reply cmd/crc/cmd/start.go]: We decided that the word "instance" works best

/cc: @robin-owen can chime in, as that is what we will do for the docs.

**Review by robin-owen — COMMENTED** [review]
My apologies, I thought I had already submitted a review of this PR, and it turns out all of my comments were pending. :man_facepalming: 

In short, I think we can replace "virtual machine" with "instance" in every case and not lose any meaning. Likewise, many times we mention the OpenShift cluster or Podman runtime we can simply use "instance" without losing meaning.

File: Makefile [hunk 5]
```diff
@@ -1,9 +1,9 @@
 SHELL := /bin/bash
 
-OPENSHIFT_VERSION ?= 4.9.12
+OPENSHIFT_VERSION ?= 4.9.15
 PODMAN_VERSION ?= 3.4.4
 BUNDLE_EXTENSION = crcbundle
-CRC_VERSION = 1.99.1
+CRC_VERSION = 1.99.2
```
Comment (cfergeau) [inline Makefile:6]: Not sure this has to be part of a bundle version updated?

Comment (praveenkumar) [reply Makefile:6]: @cfergeau no, I will remove that.

File: cmd/crc/cmd/setup_test.go [hunk 6]
```diff
@@ -14,7 +14,7 @@ func TestSetupRenderActionPlainSuccess(t *testing.T) {
 	assert.NoError(t, render(&setupResult{
 		Success: true,
 	}, out, ""))
-	assert.Equal(t, "Your system is correctly setup for using CodeReady Containers, you can now run 'crc start' to start the OpenShift cluster\n", out.String())
+	assert.Equal(t, "Your system is correctly setup for using CodeReady Containers, Run 'crc start' to start the instance\n", out.String())
```
Comment (gbraad) [inline cmd/crc/cmd/setup_test.go:17]: capital ? This then should be a full-stop

File: pkg/crc/machine/bundle/constants.go [hunk 7]
```diff
@@ -8,23 +8,23 @@ import (
 var bundleLocations = map[string]bundlesDownloadInfo{
 	"amd64": {
 		"darwin": {
-			preset.OpenShift: download.NewRemoteFile("https://storage.googleapis.com/crc-bundle-github-ci/4.9.12/crc_hyperkit_4.9.12.crcbundle",
-				"73167e735337b24efa18654667d8bff7cb2b7eb2d0bd2c6ffae4f1df4d8f6ced"),
+			preset.OpenShift: download.NewRemoteFile("https://storage.googleapis.com/crc-bundle-github-ci/4.9.15/crc_hyperkit_4.9.15.crcbundle",
+				"9e90bcbce5837cfc7c13f94d269fe043ba38c3228b57ba4748cc52d47ededf86"),
```
Comment (gbraad) [inline pkg/crc/machine/bundle/constants.go:12]: why is this part of the change?

File: test/e2e/features/basic.feature [hunk 8]
```diff
@@ -54,7 +54,7 @@ Feature: Basic test
         And stderr should contain "Checking if NetworkManager service is running"
         And stderr should contain "Using root access: Executing systemctl daemon-reload command"
         And stderr should contain "Using root access: Executing systemctl reload NetworkManager"
-        And stdout should contain "Your system is correctly setup for using CodeReady Containers, you can now run 'crc start' to start the OpenShift cluster"
+        And stdout should contain "Your system is correctly setup for using CodeReady Containers, Run 'crc start' to start the instance"
```
Comment (gbraad) [inline test/e2e/features/basic.feature:57]: Ditto. Capital?

**Review by gbraad — CHANGES_REQUESTED** [review]
Capital used after a comma

File: pkg/crc/machine/bundle/constants.go (hunk 7 above)
Comment (praveenkumar) [reply pkg/crc/machine/bundle/constants.go:12]: @gbraad it is part of different commit message to make CI happy.

Comment (gbraad) [reply pkg/crc/machine/bundle/constants.go:12]: ugh...
 OK

File: test/e2e/features/basic.feature [hunk 9]
```diff
@@ -54,7 +54,7 @@ Feature: Basic test
         And stderr should contain "Checking if NetworkManager service is running"
         And stderr should contain "Using root access: Executing systemctl daemon-reload command"
         And stderr should contain "Using root access: Executing systemctl reload NetworkManager"
-        And stdout should contain "Your system is correctly setup for using CodeReady Containers, you can now run 'crc start' to start the OpenShift cluster"
+        And stdout should contain "Your system is correctly setup for using CodeReady Containers, run 'crc start' to start the instance"
```
Comment (gbraad) [inline test/e2e/features/basic.feature:57]: Still incorrect:

"Your system is correctly setup for using CodeReady Containers. Run 'crc start' to start the instance"

File: cmd/crc/cmd/setup_test.go [hunk 10]
```diff
@@ -14,7 +14,7 @@ func TestSetupRenderActionPlainSuccess(t *testing.T) {
 	assert.NoError(t, render(&setupResult{
 		Success: true,
 	}, out, ""))
-	assert.Equal(t, "Your system is correctly setup for using CodeReady Containers, you can now run 'crc start' to start the OpenShift cluster\n", out.String())
+	assert.Equal(t, "Your system is correctly setup for using CodeReady Containers, run 'crc start' to start the instance\n", out.String())
```
Comment (gbraad) [inline cmd/crc/cmd/setup_test.go:17]: Still incorrect

"Your system is correctly setup for using CodeReady Containers. Run 'crc start' to start the instance"

**Review by gbraad — CHANGES_REQUESTED** [review]
Two sentences are concatenated with a comma. Please use a full-stop.

**Review by gbraad — APPROVED** [review]

**Pull request merged**